client/src/components/NavBar/NotificationRequest/NotificationRequest.tsx

```diff
@@ -4,28 +4,36 @@ import { Box, Typography } from '@material-ui/core';
 import useStyles from './useStyles';
 import { Notification } from '../../../interface/notification/Notification';
 import { Link } from 'react-router-dom';
+import { setUnreadNotificationToRead } from '../../../helpers/APICalls/notifications';
+import moment from 'moment';
+
 interface IProps {
   notification: Notification;
 }
 
 const NotificationRequest: React.FC<IProps> = ({
-  notification: { type, createdAt, name, image, service },
+  notification: { _id, type, createdAt, name, image },
```

Comment:
To match change in Notification interface:
```suggestion
  notification: { _id, type, createdAt, senderProfile },
```